client/src/components/sendRequsetModel/Form.tsx

```diff
@@ -0,0 +1,261 @@
+import {
+  Button,
+  FormControlLabel,
+  MenuItem,
+  Radio, RadioGroup, Select, TextField, Typography,
+} from '@mui/material';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import brands from '../../brand.json';
+
+const validationSchema = yup.object({
+  brand: yup
+    .string()
+    .required('brand is required'),
+  model: yup
+    .string()
+    .required('model is required'),
+  year: yup
+    .number()
+    .required('year is required'),
+  Mileage: yup
+    .number()
+    .required('mileage is required'),
+  price: yup
+    .number()
+    .required('price is required'),
+  location: yup
+    .string()
+    .required('location is required'),
+});
+
+function Form() {
+  const formik = useFormik({
+    initialValues: {
+      brand: '',
+      model: '',
+      year: '',
+      Mileage: '',
+      price: '',
+      location: '',
+    },
+    validationSchema,
+    onSubmit: (values) => {
+      console.log(values);
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  return (
+    <div style={{
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    }}
+    >
+      <Typography sx={{ fontSize: '1.7rem' }} component="h1">
```

Comment:
```suggestion
      <Typography sx={{ fontSize: '1.7rem' }} component="h2">
```
Change the text to Sell Your Car Now!